tox.ini

```diff
@@ -19,7 +19,7 @@ passenv = *
 
 
 deps =
-    test-{py36,py37,py38},twisted-apidoc: git+https://github.com/twisted/twisted.git
+    test-{py35,py36,py37,py38},twisted-apidoc: git+https://github.com/twisted/twisted.git
```

Comment:
It would probably be faster to use the archive URL ``https://github.com/twisted/twisted/archive/trunk.zip`` instead of a full Git clone.

Reply:
That's probably a good idea, but I'm always worried about stale cached data. Since this doesn't impact end users, I think it can be changed after the release.

Reply:
I don't think it changes anything from a caching perspective, actually. I believe that Pip does the clone in a temporary directory that gets thrown away after building the distribution. FWIW I gave up running the full suite locally because it took so long on my slow internet connection.

I agree this doesn't need to be done before the release.

Reply:
I once worked at a company that had a poorly implemented corporate proxy which was happy to serve you stale cached data unless you used every possible HTTP header to tell it not to. It made me very paranoid about caches.